src/HighlightGenerator.js

```diff
@@ -27,7 +27,7 @@ HighlightGenerator.generateHighlightNode = function generateHighlightNode() {
 
 HighlightGenerator.transformCoordinatesRelativeToParent = function transformCoordinatesRelativeToParent(rect, parentRect, scroll, fieldType) {
   var coords = {};
-  coords.top = (rect.top - parentRect.top + rect.height);
+  coords.top = (rect.top - parentRect.top + rect.height - 3);
```

Comment:
I'd like this sort of positioning adjustment to go in `HighlightGenerator.setNodeStyle` instead of `transformCoordinatesRelativeToParent`. `transformCoordinatesRelativeToParent` should be focused on converting from local coordinates to page coordinates. `setNodeStyle` should be responsible for setting the position and style of the underline element.

Reply:
Makes sense, thanks. Updated to be applied within `setNodeStyle` and set the adjustment amount as a const.